go.mod

```diff
@@ -3,10 +3,16 @@ module github.com/beyondstorage/go-service-ftp
 go 1.15
 
 require (
-	github.com/beyondstorage/go-credential v0.0.0-20210804082910-7c63d94f2826
+	github.com/beyondstorage/go-credential v0.0.0-20210918043243-9d146b959702
```

Comment:
Please use stable release of go-credential: https://github.com/beyondstorage/go-credential/releases/tag/v0.1.0